### DISCUSSION TITLE
Prefecthing example in docs confusion

### DISCUSSION THREAD

**Discussion by Phaster911** [TanStack/query#3980] [Q&A] [answered]
Hello, I've bee following the docs and I found something strange in the prefetch [example](https://tanstack.com/query/v4/docs/examples/react/prefetching), a request with key `["character", <ID>]` is made on `onMouseEnter`, however if I click on the same record a new request with the same key `["character", <ID>]` is triggered in the network tab in the devtools, is this the expected behavior? How can I prevent actually making the request when one of the characters is clicked?

Edit: please close this, the behaviour has to do with the queryclient defaults

**Comment by TkDodo** [comment, marked as answer]
because the queryKey changes, and the query has a default staleTime of zero, so you'll get a background refetch. I would use the same staleTime of 10 seconds or so on the query: https://codesandbox.io/s/hopeful-star-l1pf2y?file=/pages/index.js

feel free to PR this to the examples if you want
